packages/zosfiles/src/cli/mount/fs/fs.options.ts

```diff
@@ -0,0 +1,46 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandOptionDefinition } from "@zowe/imperative";
+
+import i18nTypings from "../../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).MOUNT.ACTIONS.FS.OPTIONS;
+
+/**
+ * Object containing all extra options to be used by the Mount API
+ */
+export const FsMountOptions: { [key: string]: ICommandOptionDefinition } = {
+    /**
+     * The file system type to mount
+     * @type {ICommandOptionDefinition}
+     */
+    fsType: {
+        name: "fs-type",
+        aliases: ["ft"],
+        description: strings.FSTYPE,
+        type: "string",
+        defaultValue: "ZFS"
+    },
+
+    /**
+     * The read/write mode for the file system
+     * @type {ICommandOptionDefinition}
+     */
+    mode: {
```

Comment:
In many cases, we don't perform validation of parameters for fear of falsely restricting something that is valid... however, in this case, the doc is pretty clear on the valid values for mode... and you have checks in your APIs to validate the values. I would add the available values on the definition.

Reply:
Added `allowedValues` to command option definition, it is also now restricted by the type `MountFsMode`